vignettes/dynami-example.R

```diff
@@ -1,150 +1,179 @@
-## ----setup, include=FALSE-----------------------------------------------------------------------------------
+## ----setup, include=FALSE----------------------------------------------------------------------------------------------
 knitr::opts_chunk$set(echo = TRUE)
 
 
-## -----------------------------------------------------------------------------------------------------------
+## ----------------------------------------------------------------------------------------------------------------------
 library(goldfish)
 data("RFID_Validity_Study")
 #?RFID_Validity_Study
 
 
-## -----------------------------------------------------------------------------------------------------------
+## ----------------------------------------------------------------------------------------------------------------------
 head(participants)
 
```

Comment:
I wonder whether we should change all these vignettes to tutorials. I think tutorials better fits how we use tutorials in courses, for example, and helps trim the size/testing burden of the package too. See [manynet](https://github.com/snlab-ch/manynet)/migraph for how we have done it there.